### ISSUE TITLE
When creating a new account from Hamburger menu the user is redirected to "Enter your password" page

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#13769]
**Build:**

Stage [Train 237.4]

**Affected platforms:**

Windows 10 64-bit

**Browsers tested on:**

Firefox, latest release

**Preconditions:**

The user doesn’t have a Fx account.

**Steps to reproduce:**

1. Launch Firefox with a new browser profile.
1. Click the Hamburger menu in the upper right hand corner and then Sign in.
1. Enter a valid email address that doesn’t have associated a Fx account and sign up.
1. Complete the account creation flow.
1. Click the “Sync Firefox across devices“ button.

**Actual result:**

The user is redirected to “Enter your password” page.

**Expected result:**

The user is redirected to “Get Firefox on your phone or tablet” page.

**Notes:**

* Might be related to [https://mozilla-hub.atlassian.net/browse/FXA-5597](https://mozilla-hub.atlassian.net/browse/FXA-5597|smart-link)
* The issue also occurs when creating account from ‘Sign in’ link in Welcome page or ‘about:preferences#sync’ page
* If entering the password again and then clicking again the “Sync Firefox across devices“ button the user is redirected correctly.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-5610)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/26874">user redirected to enter your email page1.gif</a>

**Comment by data-sync-user** [comment]
➤ Dan Schomburg commented:

Bianca Oltean We believe this is no longer an issue now that https://mozilla-hub.atlassian.net/browse/FXA-5597 ( https://mozilla-hub.atlassian.net/browse/FXA-5597|smart-link ) is fixed. Please review and re-open if this is not the case.

**Comment by data-sync-user** [comment]
➤ Bianca Oltean commented:

Dan Schomburg I verified this on stage and the issue is not reproducible anymore but now after sign-up I get to “Get Firefox on your phone or tablet“ page with the QR code. Should the “Firefox account connected“ page be displayed first?

**Comment by data-sync-user** [comment]
➤ Dan Schomburg commented:

Bianca Oltean I believe the ‘Get Firefox on your phone or tablet’ with the QR code is now the preferred page. When we addressed https://mozilla-hub.atlassian.net/browse/FXA-5452 ( https://mozilla-hub.atlassian.net/browse/FXA-5452|smart-link ), the flow was changed (note the acceptance criteria on this issue). On sign up / sign in, if there is a query parameter that is set to entrypoint=fxaappmenu, then we show the download page with the QR code.
